### ISSUE TITLE
[SUGGESTION] | Human Arthas becoming king

### ISSUE THREAD

**Issue by TheWatchingI** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#1580]
If one starts as Arthas and doesn't follow the way to become a death knight, the gameplay becomes boring rather easily. Arthas has one little county and no way to really make an impact on the war against the Scourge. Therefore I think it would be good, if there was an option for him to become king or at least gain control over a larger part of Lordaeron.

**Comment by zumbak04** [comment]
Hello, thanks for the suggestion, but I don't fully understand what you want.

> if there was an option for him to become king or at least gain control over a larger part of Lordaeron.

How? By killing his father?

**Comment by TheWatchingI** [comment]
No, I thought more about an event or something. In WC3 Arthas had a pretty important role in the war against the Scourge, but as soon as you don't follow the death knight event chain any longer, he becomes basically insignificant. 

Maybe, if Arthas isn't played by the AI and doesn't chose to become a death knight, Therenas could be killed in battle? Alternatively his father could grant him a duchy or two, if he doesn't go north, to give him a bigger force to command? Or a few event troops?

**Comment by zumbak04** [comment]
> Maybe, if Arthas isn't played by the AI and doesn't chose to become a death knight, Therenas could be killed in battle?

Could be... what an unfortunate set of circumstances...
